### ISSUE TITLE
ability to inject ca_bundles from configmap

### ISSUE THREAD

**Issue by raffaelespazzoli** [redhat-cop/cert-utils-operator#81]
ca_bundles are public info and there is a trend now make them available in configmaps rather than secrets. Cert-uitls should support injecting ca_bundles from configmaps.
A single configmap to secret injector should suffice as all other injector support secret as source.

**Comment by raffaelespazzoli** [comment]
This Fixes #41 also when closed.

**Comment by mathianasj** [comment]
Going to review if this was completed as part of or #82

**Comment by raffaelespazzoli** [comment]
@mathianasj can we close this?

**Comment by mathianasj** [comment]
I will take a look today to make sure it is present

**Comment by QuingKhaos** [comment]
Need this as well. The The Cluster Network Operator injects certificates across any namespace when an empty ConfigMap is created with the config.openshift.io/inject-trusted-cabundle=true label, but several components like the GitLab Runner Operator expect the ca-bundle within a secret.

Injecting the ca from the above configmap into the secret would be required.

**Comment by davidkarlsen** [comment]
yes, like https://docs.openshift.com/container-platform/4.14/networking/configuring-a-custom-pki.html#certificate-injection-using-operators_configuring-a-custom-pki - just targeting secrets for the storage.
